setup.go

```diff
@@ -69,6 +70,24 @@ func runSetup(yk *piv.YubiKey) {
 	if _, err := yk.Certificate(piv.SlotAuthentication); err == nil {
 		log.Println("‼️  This YubiKey looks already setup")
 		log.Println("")
+		cert, err := yk.Attest(piv.SlotAuthentication)
```

Comment:
I don't think Attest should be required. It doesn't work for imported keys (although arguably setup doesn't produce them). I recall some issues with older devices and attestation, but I forget what.

In any case, wouldn't it be better to reuse what yubikey-agent main does instead of custom logic for getting the certificate?

https://github.com/FiloSottile/yubikey-agent/blob/master/main.go#L243-L259

Reply:
You're right, thanks for the feedback. I missed that function. Fixed.